src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/BookingResourceIntTest_getOffenderAlerts.java

```diff
@@ -291,4 +291,284 @@ void canFilterBeDateCreatedAndActiveStateAndAlertType() {
             assertThat(jsonContent).extractingJsonPathBooleanValue("[0].active").isFalse();
         }
     }
+    @Nested
+    @DisplayName("GET /api/bookings/{bookingId}/alert/v2")
+    class NewUnSafeEndpoint {
```

Comment:
more or less same as above test except with new parameters and the modern paging model